recipes/magicgui/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "magicgui" %}
+{% set version = "0.1.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 65e7b6845e636a302aecf2e736232aba7f9651ea782230ab4bb58627d957e893
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install --no-deps --ignore-installed . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools_scm
+  run:
+    - python
+    - qtpy >=1.7.0
+
+test:
+  requires:
+    - pyside2
+  imports:
+    - magicgui
+
+about:
+  home: "https://github.com/napari/magicgui"
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: "build GUIs from functions, using magic."
+  doc_url: https://magicgui.readthedocs.io/en/latest/
+  dev_url: https://github.com/napari/magicgui
+
+extra:
+  recipe-maintainers:
+    - k-dominik
+    - tlambert03
```

Comment:
@tlambert03 - are you OK with being a maintainer?

Reply:
Yes, I wrote the package.  Thanks for checking!